java/Ice/greeter/serveramd/src/main/java/com/zeroc/demos/greeter/Chatbot.java

```diff
@@ -13,9 +13,9 @@
  */
 class Chatbot implements Greeter
 {
-    // Implements the abstract method Greet from the Greeter interface generated by the Slice compiler.
-    // This variant is the asynchronous implementation.
-    public CompletionStage<String>  greetAsync(String name, Current current) {
```

Comment:
It felt weird to say a method named `greetAsync` implements the `greet` method, without a little explanation.

Reply:
the method name is obviously greetAsync.

Reply:
Fixed!
And yes I agree, I mistakenly thought there was some other reason for calling it `greet`... so I kept it.
But yeah, it should definitely be `greetAsync`.